elliott/elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -261,16 +270,14 @@ def get_assembly_bug_ids(runtime, bug_tracker_type):
 
 
 def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int],
-                            permitted_bug_ids, permissive=False, major_version: int = 4):
+                            permitted_bug_ids, operator_advisory, permissive=False, major_version: int = 4):
     bugs_by_type: Dict[str, type_bug_set] = {
         "rpm": set(),
         "image": set(),
         "extras": set(),
-        # Metadata advisory will not have Bugs for z-stream releases
```

Comment:
Let's keep this comment intact for context

Reply:
added in https://github.com/openshift-eng/art-tools/commit/65be8eaf67986030464a864aa735a1cd755bfdc9